.scripts/packages.ts

```diff
@@ -242,3 +246,34 @@ async function getPackageMetadataFromReadmeFile(azureSdkForJsRoot: string, azure
         readmePath: tsReadmePath.replace(azureRestApiSpecsRoot, "")
     };
 }
+
+async function getNotBuildingPackages(azureSdkForJsRoot: string, jsonPackageInfos: PackageInfo[]): Promise<PackageFault[]> {
```

Comment:
Please rename to "getPackagesWithBuildErrors". "getNotBuildingPackages" could be interpreted to mean "packages that we aren't building for some reason".